.github/CONTRIBUTING.md

```diff
@@ -3,9 +3,21 @@
 Follow the [Meme Battle styleguide](https://github.com/Meme-Battle/styleguide/blob/main/git.md).
 
 ## Folder Structure (general) ##
-
-
-
-```
-README.md - You are here!
+``` 
+ ├── prisma/
+ │       └── migrations/ 
+ │       └── schema.prisma/ #Montagem das models
+ ├── src/
+    │   ├── providers/
+    │   ├── └── #Provedores do projeto, ex : prisma, typeorm, sequelize e etc.
+    │   ├── repositories/
+    │   ├── └── #Repositórios do projeto, ex : Onde vai estar os métodos de execução com o banco.
+    │   ├── middlewares/
+    │   │   └── #Middlewares do projeto, ex : verificadores de autênticação.
+    │   ├── modules/
+    │   │   └──<module>/
+    │   │       └── UseCases
+    │   │                 └── #Casos de uso do projeto
```

Comment:
You can create examples on folder just to other devs know about the name patterns for each type file

```suggestion
    │   │                 └── ExampleUseCase.ts #Casos de uso do projeto
```